### ISSUE TITLE
Does it support global component feature?

### ISSUE THREAD

**Issue by AbidKhairyAK** [GeekyAnts/vue-native-core#154]
Does vue native support global component feature?

i've tried this code but seems it doesn't work:

```
import MyComponent from './MyComponent';

Vue.component('my-component', MyComponent);
```

**Comment by icarter09** [comment]
@AbidKhairyAK were you able to figure this issue out or solve it?

**Comment by AbidKhairyAK** [comment]
NO,
I have not been able to solve this problem. until now, I still import custom components on every file that needs it

**Comment by AbidKhairyAK** [comment]
@icarter09 NO,
I have not been able to solve this problem. until now, I still import custom components on every file that needs it

**Comment by icarter09** [comment]
@AbidKhairyAK thanks for heads up. I'll try to recreate the issue and dig into it.

**Comment by icarter09** [comment]
@AbidKhairyAK I don't believe this is supported. `Vue.component` is not being picked up on as valid syntax when I tested it. I would put this down as a feature request or enhancement.

@sanketsahusoft can you add the 'Enhancement' label or 'Feature Request' label to this ticket? Thanks.

**Comment by crolly** [comment]
Hi @AbidKhairyAK and @icarter09 ,

since I ran into the same issue today, I came up with quite a simple workaround just using the Vue plugin capability.
To register components globally, you just need to write your own little plugin like this:

```javascript
import componentA from './componentA'

export default {
  // The install method is all that needs to exist on the plugin object.
  // It takes the global Vue object as well as user-defined options.
  install(Vue) {
    Vue.component('component-name', componentA)
  }
}
```

In your app entry file you can just use that plugin like this:

```javascript
import Vue from 'vue-native-core'

import components from './src/components'
Vue.use(components)
```

I hope this helps, although the answer might have come a little late.
